stacker/blueprints/asg.py

```diff
@@ -165,9 +165,18 @@ def create_load_balancer(self):
                     GetAtt(elb_name, 'DNSName')],
                 Condition="SetupELBDNS"))
 
+    def get_launch_configuration_parameters(self):
```

Comment:
With the way parameters are handled now you shouldn't have to copy/paste.  Subclass, then modify _get_parameters.  If you have an idea for a standard that works for everything, I'm all for it.